src/ResourceManagement/CosmosDB/CassandraKeyspaceImpl.cs

```diff
@@ -0,0 +1,233 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace Microsoft.Azure.Management.CosmosDB.Fluent
+{
+    using Microsoft.Azure.Management.CosmosDB.Fluent.Models;
+    using Microsoft.Azure.Management.ResourceManager.Fluent.Core;
+    using System.Collections.Generic;
+    using System.Threading;
+    using System.Threading.Tasks;
+
+    internal partial class CassandraKeyspaceImpl :
+        ExternalChildResource<ICassandraKeyspace,
+            CassandraKeyspaceGetResultsInner,
+            ICosmosDBAccount,
+            CosmosDBAccountImpl>,
+        ICassandraKeyspace,
+        CassandraKeyspace.Definition.IDefinition<CosmosDBAccount.Definition.IWithCreate>,
+        CassandraKeyspace.Update.IUpdate,
+        CassandraKeyspace.Definition.IDefinition<CosmosDBAccount.Update.IWithOptionals>
+    {
+        private ICassandraResourcesOperations Client { get { return Parent.Manager.Inner.CassandraResources; } }
+        private CassandraKeyspaceCreateUpdateParameters createUpdateParameters;
```

Comment:
Maybe I only missed it. I didn't find initialization properties of createUpdateParameters to existing setting when Update().

Reply:
function name: InitializeCreateUpdateParameters